lib/ssl/src/tls_handshake_1_3.erl

```diff
@@ -79,7 +79,7 @@ server_hello(MsgType, SessionId, KeyShare, PSK, ConnectionStates) ->
     #{security_parameters := SecParams} =
 	ssl_record:pending_connection_state(ConnectionStates, read),
     Extensions = server_hello_extensions(MsgType, KeyShare, PSK),
-    #server_hello{server_version = {3,3}, %% legacy_version
+    #server_hello{server_version = ?'TLS-1.2', %% legacy_version
```

Comment:
Here we probably should use the legacy-macro!

Reply:
what legacy macro?

Reply:
?LEGACY_VERSION from tls_record_1_3.hrl